### PR TITLE
Completely Remove Old Style Scene

### PR DESCRIPTION
**Pull request by Rheeseyb**
Follow up from #1083 

The previous PR introduced all of the functionality to allow us to bin off the old style scenes, but stopped short of removing it from the tests. This PR updates all default projects to use the natural scene style, removes the now dead `componentName` prop from the metadata, and updates all scenes to use the natural scene style. It's a monster because of the number of tests that required updating to finish this off.

The only "functional" changes in here are the removal of `componentName` (and its uses in `element-metadata-utils.ts`), the updating of default scenes to use the new natural scenes, and the removal of `SceneRootRenderer`. Everything is a consequential test update.

Upon approval, this will be merged into #1083 before that PR is merged to master.

### CODE REVIEW

**Review by enidemi — APPROVED** [review]

File: editor/src/components/editor/store/editor-state.spec.ts [hunk 1]
```diff
@@ -127,9 +132,28 @@ describe('modifyUnderlyingTarget', () => {
...
+                left: 0,
+                top: 0,
+                width: 50,
+                height: 50,
+                backgroundColor: 'red',
+              }}
+              data-thing='a thing'
+            />
+            <Rectangle
+              style={{
```
Comment (balazsbajorics) [inline editor/src/components/editor/store/editor-state.spec.ts:148]: this style prop is new here, right? where is this coming from?

Comment (Rheeseyb) [reply editor/src/components/editor/store/editor-state.spec.ts:148]: The tests were previously using a different version of the complex multifile project for no apparent reason, so I unified that to ensure we kept the two inline (as I was updating the sample project and wondering why it didn't break multifile tests)

File: editor/src/test-cases/majestic-broker.ts [hunk 2]
```diff
@@ -437,27 +437,18 @@ export var App = () => {
 }
 export var storyboard = (
   <Storyboard>
-    <Scene
-      component={App}
-      props={{}}
-      style={{ position: 'absolute', left: 0, top: 0, width: 522, height: 652 }}
-    />
-    <Scene
-      data-label={'ScrollablePhotoGrid'}
```
Comment (balazsbajorics) [inline editor/src/test-cases/majestic-broker.ts:446]: did scenes lose their data-label?

**Review by balazsbajorics — APPROVED** [review]
my brain is blunt now

File: editor/src/test-cases/majestic-broker.ts (hunk 2 above)
Comment (Rheeseyb) [reply editor/src/test-cases/majestic-broker.ts:446]: Nope, this one was accidental

**Pull request merged**